systemverilog-plugin/Makefile

```diff
@@ -27,31 +27,23 @@ SOURCES = UhdmAst.cc \
 # Directory to search for Surelog and UHDM libraries
 UHDM_INSTALL_DIR ?= /usr/local
 
+# Tell pkg-config to look in the provided install path first.
+# PKG_CONFIG_PATH and PKG_CONFIG_PATH_FOR_TARGET are search paths it looks in
+# so set the environment variables and prefix with our local install first
+PKG_CONFIG_INVOKE=\
+   PKG_CONFIG_PATH=$(UHDM_INSTALL_DIR)/lib/pkconfig:${PKG_CONFIG_PATH} \
+   PKG_CONFIG_PATH_FOR_TARGET=$(UHDM_INSTALL_DIR)/lib/pkconfig:${PKG_CONFIG_PATH_FOR_TARGET} \
```

Comment:
```suggestion
PKG_CONFIG_INVOKE = \
   PKG_CONFIG_PATH=$(UHDM_INSTALL_DIR)/lib/pkgconfig:${PKG_CONFIG_PATH} \
   PKG_CONFIG_PATH_FOR_TARGET=$(UHDM_INSTALL_DIR)/lib/pkgconfig:${PKG_CONFIG_PATH_FOR_TARGET} \
```

Reply:
LOL, I wonder how that worked on my machine. I think I had set the environment variable in my shell already, so that the second part of the path worked.
Fixed.